13-loops/exercises/04-only-evens/main.go

```diff
@@ -28,4 +26,50 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	args := os.Args[1:]
+	if len(args) < 2 {
+		fmt.Println("Please pass min and max integers.")
+		return
+	}
+
+	min, err := strconv.Atoi(args[0])
+
+	if err != nil {
+		fmt.Printf("Either %q or %q is not an integer.\n", args[0], args[1])
+	}
+
+	max, err := strconv.Atoi(args[1])
+
+	if err != nil {
+		fmt.Printf("Either %q or %q is not an integer.\n", args[0], args[1])
+	}
+
+	if min > max {
+		fmt.Printf("The max integer is less than the min integer.\n")
+		return
+	}
+
+	var sum int
+	for i := min; i <= max; i++ {
+		isEven := i%2 == 0
+		if i != max && isEven {
+			fmt.Printf("%d ", i)
+
+			if i != max-1 {
+				fmt.Printf("+ ")
+			}
+		}
+
+		// sum even numbers
+		if isEven {
```

Comment:
you should use 'continue' to skip the odd numbers

Reply:
Fixed